hack/ci/e2e-conformance.sh

```diff
@@ -222,6 +222,8 @@ init_image() {
       rm packer.zip && \
       ln -s "$PWD"/packer /usr/local/bin/packer
   fi
+
+  if [[ -n ${CI_VERSION:-} ]]; then
```

Comment:
do we need the `:-`? My bash-fu isn't great. I thought it's typically used for defaulting

Reply:
Yes, without it we would error if the variable is unset (because of the set command on line 19 that enables `-o nounset`)